### PR TITLE
Added alerts documentation

### PR DESCRIPTION
**Pull request by ColRad**

### CODE REVIEW

File: docs/product-manuals/cloud-console/manage-clusters/manage-alerts.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+To create a new alert you have to navigate into the `Alert` tab:
+
+![cluster-details](../../../guides/getting-started/img/cluster-detail-alerts.png)
+
+Click **Create** to create a new alert.
+
+![create-alert](../../../guides/getting-started/img/cluster-detail-create-alert.png)
+
+You can choose between `Email` and `Webhook`. For `Email` no further information is needed.
+To create a `Webhook` alert, you need to provide a valid webhook url that excepts `POST` requests.
```
Comment (ultraschuppi) [inline docs/product-manuals/cloud-console/manage-clusters/manage-alerts.md:22]: and HEAD right?

Comment (ColRad) [reply docs/product-manuals/cloud-console/manage-clusters/manage-alerts.md:22]: well if it accepts POST it accepts HEAD

**Comment by ultraschuppi** [comment]
thanks @ColRad 
@akeller wanna have a look over it as well? will get merged in 2nd week of march

**Review by ultraschuppi — APPROVED** [review]

**Comment by akeller** [comment]
> thanks @ColRad
> @akeller wanna have a look over it as well? will get merged in 2nd week of march

2nd week of May? 😄

**Comment by ultraschuppi** [comment]
> 2nd week of May?

oh dear... yes ;)

**Comment by ultraschuppi** [comment]
@ColRad i think the images should rather reside in `product-manuals/cloud-console/img`, but since they are having a few pixels of decorations anyway - what about i redo them? ;)

**Comment by akeller** [comment]
Is this merging into `master`?  If so, please remove references to "workflow". Process -> workflow changes are already staged in `release_0_27` including the variables names for the code snippet.

"Alerts" should be lower case for all references except when quoting the UI.

**Comment by ultraschuppi** [comment]
> Is this merging into `master`? If so, please remove references to "workflow". Process -> workflow changes are already staged in `release_0_27` including the variables names for the code snippet.

we will just merge this into master as soon as the new release hits the stage (and afterwards cherrypick the mergecommit to the corresponding RB)

> 
> "Alerts" should be lower case for all references except when quoting the UI.

thanks! @ColRad made the corresponding changes

File: docs/product-manuals/cloud-console/introduction.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ The Cloud Console is the management application for the included products. Via t
 
 - [create](./manage-clusters/create-cluster.md) and [delete](./manage-clusters/delete-cluster.md) clusters,
 - [manage API Clients](./manage-clusters/manage-api-clients.md) to interact with Zeebe and Tasklist,
+- [manage Alerts](./manage-clusters/manage-alerts.md) to get notified when workflow error occur,
```
Comment (akeller) [inline docs/product-manuals/cloud-console/introduction.md:10]: lowercase "alerts"

**Review by akeller — CHANGES_REQUESTED** [review]
Small typo, otherwise this looks magnificent! Thank you both!

**Review by akeller — COMMENTED** [review]
This is asking me for another review, so here it is! This looks good.

**Comment by urbanisierung** [comment]
replaced by https://github.com/camunda-cloud/camunda-cloud-documentation/pull/233

**Comment by ultraschuppi** [comment]
replaced by #233

**Pull request closed without merging**